### PR TITLE
fix: apply multiaddr patch

### PR DESCRIPTION
**Pull request by smrz2001**

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
   },
   "scripts": {
     "test": "NODE_OPTIONS=--experimental-vm-modules npx jest --silent --coverage --runInBand --forceExit",
+    "postinstall": "shx cp -r ../../node_modules/multiaddr node_modules/multiaddr && patch-package",
```
Comment (smrz2001) [inline packages/cli/package.json:36]: There are 2 other packages that use the outdated IPFS HTTP client - `ipfs-daemon` and `pinning-ipfs-backend`. IMO, only the Ceramic CLI has the crash vulnerability, but I can also patch the others to be safe. WDYT?

Comment (stbrody) [reply packages/cli/package.json:36]: I guess if you test with the bundled ipfs and do some pinning/unpinning requests and it all works then it's probably fine?

**Review by stbrody — APPROVED** [review]
I can see the intent and it seems reasonable, but will have to defer to @ukstv on if the changes here actually do what they intend in a good way

**Review by ukstv — APPROVED** [review]
If it works, it is approved.

Seems legit/innocent enough to me.

**Pull request merged**